src/AlpineSkiHouse.Web/Startup.cs

```diff
@@ -13,6 +13,8 @@
 using AlpineSkiHouse.Models;
 using AlpineSkiHouse.Services;
 using Microsoft.AspNetCore.Mvc;
+using MediatR;
+using Scrutor;
```

Comment:
It’s a package that does assembly scanning for aspnetcore di. I got it from the mediatr sample